projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-checkout.ts

```diff
@@ -0,0 +1,242 @@
+      cy.getByText('Account');
+    });
+  cy.get('.cx-review-summary-card')
+    .contains('cx-card', 'Cost Center')
+    .find('.cx-card-container')
+    .within(() => {
+      cy.getByText(costCenter);
+      cy.getByText(`(${b2bUnit})`);
+    });
+  cy.get('.cx-review-summary-card')
+    .contains('cx-card', 'Ship To')
+    .find('.cx-card-container')
+    .within(() => {
+      cy.getByText(sampleUser.fullName);
+      cy.getByText(sampleUser.address.line1);
+    });
+  cy.get('.cx-review-summary-card')
+    .contains('cx-card', 'Shipping Method')
+    .find('.cx-card-container')
+    .within(() => {
+      cy.getByText('Standard Delivery');
+    });
+  cy.get('cx-order-summary .cx-summary-row .cx-summary-amount')
+    .eq(0)
+    .should('contain', cartData.total);
+  cy.get('cx-order-summary .cx-summary-row .cx-summary-amount')
+    .eq(1)
+    .should('contain', cartData.estimatedShipping);
+  cy.get('cx-order-summary .cx-summary-total .cx-summary-amount').should(
+    'contain',
+    cartData.totalAndShipping
+  );
+  cy.getByText('Terms & Conditions')
+    .should('have.attr', 'target', '_blank')
+    .should(
+      'have.attr',
+      'href',
+      `/${Cypress.env('BASE_SITE')}/en/USD/terms-and-conditions`
+    );
+  cy.get('input[formcontrolname="termsAndConditions"]').check();
+  const orderConfirmationPage = waitForPage(
+    '/order-confirmation',
+    'getOrderConfirmationPage'
+  );
+  cy.get('cx-place-order button.btn-primary').click();
+  cy.wait(`@${orderConfirmationPage}`).its('status').should('eq', 200);
+}
+
+export function reviewB2bOrderConfirmation(
+  sampleUser: SimpleUser = b2bAccountShipToUser,
+  sampleProduct: SampleProduct = b2bProduct,
+  cartData: SampleCartProduct,
+  isAccount: boolean = true
+) {
+  cy.get('.cx-page-title').should('contain', 'Confirmation of Order');
+  cy.get('h2').should('contain', 'Thank you for your order!');
+  cy.get('.cx-order-review-summary .container').within(() => {
+    cy.get('.summary-card:nth-child(1)').within(() => {
+      cy.get('cx-card:nth-child(1)').within(() => {
+        cy.get('.cx-card-title').should('contain', 'Order Number');
+        cy.get('.cx-card-label').should('not.be.empty');
+      });
+      cy.get('cx-card:nth-child(2)').within(() => {
+        cy.get('.cx-card-title').should('contain', 'Placed on');
+        cy.get('.cx-card-label').should('not.be.empty');
+      });
+      cy.get('cx-card:nth-child(3)').within(() => {
+        cy.get('.cx-card-title').should('contain', 'Status');
+        cy.get('.cx-card-label').should('not.be.empty');
+      });
+    });
+    cy.get('.summary-card:nth-child(2) .cx-card').within(() => {
+      cy.contains(poNumber);
+      if (isAccount) {
+        cy.contains('Account');
+        cy.contains(costCenter);
+        cy.contains(`(${b2bUnit})`);
+      } else {
+        cy.contains('Credit Card');
+      }
+    });
+    cy.get('.summary-card:nth-child(3) .cx-card').within(() => {
+      cy.contains(sampleUser.fullName);
```

Comment:
just want to point out some things that I finally noticed that was done in original files for checkout are

cy.contains and cy.getByText just gets the text but it does not assert anything. HOWEVER, I think it's fine since it checks if any of those things that you passed to it exist. 

I am just saying maybe it would have been best if it had a `should`, but it's fine as that's how it was done before